hermes_eea/calibration/build_spectra.py

```diff
@@ -0,0 +1,57 @@
+
+from hermes_core.timedata import HermesData
+import astropy.units as astropy_units
+from astropy.timeseries import TimeSeries
+from astropy.time import Time
+from hermes_core.timedata import HermesData
+from astropy.nddata import NDData
+from ndcube import NDCube, NDCollection
+from hermes_eea.util.time.iso_epoch import epoch_to_iso_obj, epoch_to_eea_iso, epoch_to_iso
+import numpy as np
+from astropy.wcs import WCS
+
+class Build_Hermes_EEA_Data:
```

Comment:
This class name starts with a verb which suggests that it would be better if it were a function. Also missing doc strings.